src/main/java/org/example/daos/JobRoleDao.java

```diff
@@ -52,7 +52,7 @@ public JobRole getJobRoleById(final int id,
                         + "capabilityID, bandID, "
                         + "closingDate, status, description, "
                         + "responsibilities, jobSpec FROM `Role` "
-                        + "WHERE id=?;";
+                        + "WHERE id=? AND status='open';";
```

Comment:
not needed, the AC doesn't specify open roles

Reply:
but the testers were complaning that you could:
You can access closed jobs by entering their ID in the URL's 
so adding a status open wont allow this or should we get the frontend to account for this?

Reply:
neither. its fine to see the details for closed roles. they just shouldn't appear on the list page. the acceptance criteria reflect this